web/html/javascript/ajax.js

```diff
@@ -0,0 +1,20 @@
+var onAjaxRequestError = function (res, status, error) {
+  console.log("Error processing ajax request...")
```

Comment:
Adding a comment that this is temporary would be nice. I just hope it'll be gone before 10 years